dcos/dcos-ui
# [marathon 1.5] DCOS-14703: UCR Improvements

How to test this:

Launch a cluster with a custom configuration specifying **`testing/pull/1481`** to use the snapshot release https://github.com/dcos/dcos/pull/1481/files 

This release though doesn't accept the old definition of a BRIDGE network for UCR, 
it supports the new version instead: 

```javascript
{
  "networks": [
    {
      "mode": "container/bridge"
    }
  ]
}
```

and it returns updated definition in the new format.

Host volumes and force Pull Image work as expected.

Having said that we've concluded to run this PR against the integration branch `integrate/marathon-1.5` that will be merged to master once all the respective changes are merged in Marathon/DCOS.
